### DISCUSSION TITLE
Export Forms

### DISCUSSION THREAD

**Discussion by rspilhaus** [solspace/craft-freeform#839] [Feature Requests]
Hello! I think it would be useful to have the ability to export the forms themselves (not just the submissions).

Often I'll create something locally or on a dev site and then need to recreate it on prod. 

A good example of this functionality is Gravity Forms for WordPress.

**Comment by kjmartens** [comment]
Thank you for your feedback. This is a planned feature in Freeform 5 (now in beta). 🙂

**Reply by rspilhaus** [reply]
Exciting - thanks!

**Reply by kjmartens** [reply]
For sure! I'm going to leave it open for now so others can see the status of it. 🙂

**Comment by kjmartens** [comment]
This has now been added in Freeform 5.6+ 🙂 

It now includes a utility for granularly exporting and importing Freeform data (beta), including forms, submissions, templates, integrations, and Freeform settings.
